MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -909,31 +987,36 @@ private void initMenu() {
         //endregion View Menu
 
         //region Manage Campaign Menu
+        // The Manage Campaign menu uses the following Mnemonic keys as of 19-March-2020:
+        // A, B, G, M, S
         JMenu menuManage = new JMenu(resourceMap.getString("menuManageCampaign.text"));
+        menuManage.setMnemonic(KeyEvent.VK_A);
         menuManage.setName("manageMenu");
 
         JMenuItem miGMToolsDialog = new JMenuItem(resourceMap.getString("miGMToolsDialog.text"));
-        miGMToolsDialog.setEnabled(true);
+        miGMToolsDialog.setMnemonic(KeyEvent.VK_G);
         miGMToolsDialog.addActionListener(evt -> showGMToolsDialog());
         menuManage.add(miGMToolsDialog);
 
-        JMenuItem miAdvanceMultipleDays = new JMenuItem(resourceMap.getString("miAdvanceMultipleDays.text"));
-        miAdvanceMultipleDays.setEnabled(getCampaignController().isHost());
-        miAdvanceMultipleDays.addActionListener(evt -> showAdvanceDaysDialog());
-        menuManage.add(miAdvanceMultipleDays);
+        if (getCampaignController().isHost()) {
```

Comment:
?

Reply:
I'm hiding the menu item if it can't be shown, unlike the current variant of disabling it

Reply:
As this is never changed after the menu is enabled, this is the better way to implement it

Reply:
Actually, thinking more about it this should be visible based on the current host, and enabled when that changes. I'll set that up now.

Reply:
Switched to enabling the visual based on whether the campaign is the host or not, and added a method to switch it out. However, there is no obvious way to swap based on the current CampaignController code